src/main/kotlin/ui/OutputFormatSelector.kt

```diff
@@ -133,6 +134,7 @@ class OutputFormatSelector : RComponent<OutputFormatSelectorProps, RState>() {
 
     private val Format.description: String?
         get() = when (this) {
+            Midi -> Strings.MIDIFormatDescription
```

Comment:
ditto.
`VocalodiMidFormatDescription`
The others should also be like `VsqxFormatDescription` (I missed these items when changing the overall naming style.